arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/stream/TerminalOpsShortcuts.kt

```diff
@@ -0,0 +1,7 @@
+package arrow.fx.coroutines.stream
+
+/**
+ * This is a shortcut for [compile] + [TerminalOps.forEach]
+ */
+suspend fun <O> Stream<O>.forEach(block: (O) -> Unit): Unit =
```

Comment:
The design of `TerminalOps` was made to make all terminal operators very explicit, since this is where you exit the streaming world. Which you should typically never do this in functional streaming applications since you want to use `Stream` all the way to the edge in those cases.

Adding this would nullify the whole design of `TerminalOps`, and `TerminalResourceOps`.

Reply:
Good point, I'll remove the shortcut 😁